controllers/selfnoderemediation_controller.go

```diff
@@ -624,3 +630,28 @@ func (r *SelfNodeRemediationReconciler) restoreNode(nodeToRestore *v1.Node) (ctr
 	// all done, stop reconciling
 	return ctrl.Result{Requeue: true}, nil
 }
+
+func (r *SelfNodeRemediationReconciler) updateSnrStatusLastError(snr *v1alpha1.SelfNodeRemediation, err error) error {
+	var lastErrorVal string
+
+	if err != nil {
+		lastErrorVal = err.Error()
+	} else {
+		lastErrorVal = "successful reconcile"
```

Comment:
According to the field description in the CRD it should be empty when no error occured. Which makes more sense to me than this message.